openlibrary/plugins/openlibrary/js/ol.js

```diff
@@ -1,3 +1,60 @@
+var Carousel = {
+    add: function(selector, a, b, c, d, e, f) {
```

Comment:
Fine for them to be concise, fine for them to have number suffixes, but we need better parameter names than "a, b, c, d, e, f" :) (...unless this signature is being dictated from elsewhere, in which case a comment would be helpful to explain)